app/setup.py

```diff
@@ -342,6 +345,24 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_task_client(application):
+    if application.config["EQ_SUBMISSION_CONFIRMATION_BACKEND"] == "cloud-tasks":
+        application.eq["task-client"] = CloudTaskPublisher()
+
+        if not application.config["EQ_SUBMISSION_CONFIRMATION_QUEUE"]:
+            raise Exception("Setting EQ_SUBMISSION_CONFIRMATION_QUEUE Missing")
+
+        if not application.config["SUBMISSION_CONFIRMATION_CLOUD_FUNCTION_NAME"]:
```

Comment:
Given `EQ_SUBMISSION_CONFIRMATION_QUEUE` is defaulted, I think `SUBMISSION_CONFIRMATION_CLOUD_FUNCTION_NAME` should also be defaulted and removed fromt this check.